src/pudl/transform/ferc714.py

```diff
@@ -1147,7 +1148,10 @@ def average_duplicate_pks_csv(df):
             & (df["forecast_year"] == 2014)
             & (df["net_demand_forecast_mwh"] == 0)
         )
-        assert len(df[error_mask] == 1)
+        if (len_dupes := len(df[error_mask])) >= 1:
+            raise AssertionError(
+                f"We found {len_dupes} duplicate errors, but expected 1 or less:\n{df[error_mask]}"
```

Comment:
Does this message line up with the if statement?  The if says that finding 1 is not acceptable, but the message seems to say that 1 is acceptable.

Reply:
it says one or less

Reply:
one is acceptable

Reply:
But I think the assertion will be raised if it's 1, because the if says `>= 1`. I think it should probably be `> 1`?

Reply:
yes you are right ty!

Reply:
fixed